### PR TITLE
feat: GCED2011

### LINKED ISSUE #52: feat: add GCED2011 to NACE2 concordance
should be the same up to the 4th digit
http://www.stat.kg/stat.files/class.files/%D0%BF%D1%80%D0%BE%D0%B5%D0%BA%D1%82%20%D0%93%D0%9A%D0%AD%D0%94-3.pdf

Table version of standard can be extracted from here:
https://web.archive.org/web/20201006225214/http://stat.kg/media/files/37dc1a99-f065-463f-8f12-6ba7ad23607e.DOC

### PR DESCRIPTION
**Pull request by nyoungstudios**
## Description
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
- adds GCED2011 codes and mapping to NACE2
- closes #52 

## Types of changes

What types of changes does your code introduce:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: pyisic/_standards/gced2011/gced2011_to_nace2.py [hunk 1]
```diff
@@ -0,0 +1,1008 @@
+"""`Concordance between GCED2011 and NACE2 <http://www.stat.kg/stat.files/class.files/%D0%BF%D1%80%D0%BE%D0%B5%D0%BA%D1%82%20%D0%93%D0%9A%D0%AD%D0%94-3.pdf>_`.
+
+All GCED2011 codes that are 4 digits long map directly to the same NACE2 code.
```
Comment (JSPromisel) [inline pyisic/_standards/gced2011/gced2011_to_nace2.py:3]: Is this really true with concordances like L23? That seems to be 3 digits. Can we change the language or confirm if that mapping is valid?

**Review by JSPromisel — CHANGES_REQUESTED** [review]
Just one comment

File: pyisic/_standards/gced2011/gced2011_to_nace2.py (hunk 1 above)
Comment (nyoungstudios) [reply pyisic/_standards/gced2011/gced2011_to_nace2.py]: whoops, I meant 4 or less

**Review by JSPromisel — APPROVED** [review]
LGTM. Once this is merged, I will publish a new version to PyPI.

**Review by JSPromisel — APPROVED** [review]

**Pull request merged**